package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@reduxjs/toolkit": "^1.9.5",
```

Comment:
- [ ] Good job so far, I would implore you to add the react-redux dependency too, just as required as you will use it later.

![image](https://github.com/Daniel-Kigozi/react-book2/assets/86200356/26d3cb7c-59a3-4e0b-b2c5-3fdfd251289b)